libs/common/NumUtils.cs

```diff
@@ -75,6 +76,80 @@ public static unsafe void LongToBytes(long value, int length, ref byte* result)
             result += length;
         }
 
+        /// <summary>
+        /// Convert double number into sequence of ASCII bytes
+        /// </summary>
+        /// <param name="value">Value to convert</param>
+        /// <param name="dest">Span Byte</param>
+        /// <returns>Length of number in result</returns>
+        public static int DoubleToSpanByte(double value, Span<byte> dest)
+        {
+            int totalLen = NumOfCharInDouble(value, out var integerDigits, out var signSize, out var fractionalDigits);
+            bool isNegative = value < 0;
+            if (totalLen > dest.Length)
+                return 0;
+            fixed (byte* ptr = dest)
+            {
+                byte* curr = ptr;
+                DoubleToBytes(value, integerDigits, fractionalDigits, ref curr);
+            }
+
+            return totalLen;
+        }
+
+        /// <summary>
+        /// Convert double into sequence of ASCII bytes
+        /// </summary>
+        /// <param name="value">Double value to convert</param>
+        /// <param name="length">Number of digits in value</param>
```

Comment:
Fix the warnings shown at https://github.com/microsoft/garnet/pull/699/files

Reply:
Fixed